### DISCUSSION TITLE
Issue with naming RPM packages for EL9

### DISCUSSION THREAD

**Discussion by venice77** [rabbitmq/rabbitmq-server#8402] [Other] [answered]
### Describe the bug

Hello RabbitMQ team.

I have recently noticed that packages dedicated for EL9 are named for EL8

This is a screen from https://cloudsmith.io/~rabbitmq/repos/rabbitmq-server/packages/?q=distribution%3Ael+AND+distribution%3A9

![image](https://github.com/rabbitmq/rabbitmq-server/assets/54037365/386d285b-cc88-4d4b-9385-95c3a51918f7)

The packaged then shows in RedHat 9 as a dedicated for el8

![image](https://github.com/rabbitmq/rabbitmq-server/assets/54037365/a172fc1f-4a9f-4635-bd07-cf4e492845e8)

The naming issue shows on all mirrors, including cloudsmith.io and packagecloud.io

### Reproduction steps

1. Build new package for EL9

### Expected behavior

The rpm package show be dedicated to EL9 i.e. for Erlang 26

![image](https://github.com/rabbitmq/rabbitmq-server/assets/54037365/827a34f8-eff9-4f51-b34f-c7e7734db5d0)

**Comment by michaelklishin** [comment, marked as answer]
It is the same package, there is no difference between CentOS Stream 8 and CentOS Stream 9 that would matter for RabbitMQ. Our zero dependency Erlang RPM has a separate package for CentOS Stream 9 but even then, the "el8" version will work just fine.

We won't be introducing "el9" packages just yet. Our team is small and we have significantly more
important things to worry about.
